content/get-haiku/burn-cd.html

```diff
@@ -1,22 +1,25 @@
 +++
 type = "article"
-title = "Burning the Haiku CD"
+title = "Burning the Haiku DVD"
 date = "2009-09-11T16:04:54.000Z"
 tags = []
 +++
 
-<div class="intro">To create your Haiku Installation CD, you need to burn the ISO or Anyboot image to a CD. There are many different ways to burn an them to CD. Shown below is a list of links to how to guides and/or CD burning applications for various different operating systems:</div>
+<div class="intro">To create your Haiku Installation DVD, you need to burn the ISO or Anyboot image to a DVD. There are many different ways to burn an them to DVD. Shown below is a list of links to how to guides and/or DVD burning applications for various different operating systems.
+</div>
+
+<p>If you'd like to create your own DVD cover, we have instructions on how to <a href="/get-haiku/cd-paper-sleeve.html">print out and fold your own sleeve</a>.</p>
 
 <div class="alert alert-warning">
-<strong>A word of caution on burning CD images:</strong>
+<strong>A word of caution on burning DVD images:</strong>
 Some disc burning software may try to be "clever" and won't burn the Anyboot image properly. The following image burning applications are reported to work properly:
 <ul>
 <li>Burn 2.4.1u</li>
 <li>CDRecord</li>
 <li>InfraRecorder</li>
 <li>K3b</li>
 </ul>
-<strong>Using CDBurnerXP 4.3.2.2212 must currently be advised against</strong>, as it reportedly has produced faulty CDs from the ISO images.
+<strong>Using CDBurnerXP 4.3.2.2212 must currently be advised against</strong>, as it reportedly has produced faulty DVDs from the ISO images.
```

Comment:
They are at version 4.5.something now. Did they fix the problems? And do we need to list this many tools when just one would work?

Reply:
I'm not a Windows user, I don't know about v4.5.
I could reduce the list of Windows apps to "InfraRecorder", which is also listed in the top info box as working.

Reply:
Yes, InfraRecorder is the way to go on Windows.

Reply:
Alright. Only keeping InfraRecorder.